tfx/utils/path_utils.py

```diff
@@ -72,11 +67,32 @@ def serving_model_dir(output_uri: Text) -> Text:
 
 
 def serving_model_path(output_uri: Text) -> Text:
```

Comment:
merger serving_model_working_path in to serving_model_path

the functionality for serving_model_path should be `get the pure model path no matter the structure`